decidim-core/db/seeds.rb

```diff
@@ -22,15 +22,13 @@
   )
 
   3.times.each do
-    Decidim::Scope.create!(
-      name: Faker::Address.unique.state,
+    Decidim::Scope.find_or_initialize_by(name: Faker::Address.unique.state).update!(
```

Comment:
Yes. Seeding a second time will create 3 new scopes. And so on. This change is not actually necessary since the name is unique already.